generate_payload

```diff
@@ -158,7 +160,7 @@ BuSXfFa70kiZzpwKMP/jB8ylWdA74VzkCFfYcdwJHzzrcDS64VRqNhWM/vRFJmLP
 wN4MHkAE5RDb4cjGAwkwmZQuDzuk2O9oOukxKd7v/ZUmql4k0qDxi3M9dC3SJJ+O
 fVPRlyZ74UVlspgjr5zxSBCerj/aDbVSWWr6JjgeRTQdg6WKhO0+mfmttiANxv/a
 fBMDaxys9ee5sJL+WHP62fucD8ukmMEVM0P971U/JBfV8r8VRpy+OENgt6ynJ9dV
-4YCdOT2xo42YwkBCYcVOF6iY2YqFd3oDSZARqEk4vr+A2/eNDU37+OBWr8E1pfO7
+4YCdOT2xo42YwkBCYcVOF6iY2YqFdoDSZARqEk4vr+A2/eNDU37+OBWr8E1pfO7
```

Comment:
Not sure why that changes here.
Will leave the PR open and have a closer look next week.

Reply:
That looks like a copy+paste; the "3" fell off? I don't think it should be there. In any case, great catch, thank you Dongsu!

Reply:
Fixed, thanks again!